trackReps/include/RKTrackRep.h

```diff
@@ -284,9 +284,13 @@ class RKTrackRep : public AbsTrackRep {
 
   double momMag(const M1x7& state7) const;
 
+ protected:
 
```

Comment:
These two members are used in the `Streamer` workaround, so need to make them non-private in the `MplTrackRep`.
There is a chance though, that I am doing something stupid with this whole ROOT `ClassDef` business as don't understand it a 100%.

Reply:
Hm, I am not sure. So yes, if you reimplement the Streamer those have to be protected. But I am not sure you will need to do it anyways. Did you try without a specific implementation of a streamer? Because in principle ROOT should be "smart" enough to build on for you.

Reply:
The base `RKTrackRep` has reimplemented `Streamer` as it 
> works around a msvc bug

and so I felt I need to do so too.  
If I just remove the implementation, I get an undefined reference from the linker.